src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -188,138 +193,11 @@ internal static int GetXmlDsigSearchDepth()
             //Keeping the default recursion limit to 20. It should be
             //within limits of real world scenarios. Keeping this number low
             //will preserve some stack space
-            long maxXmlDsigSearchDepth = GetNetFxSecurityRegistryValue("SignedDigitalSignatureXmlMaxDepth", 20);
 
-            s_xmlDsigSearchDepth = (int)maxXmlDsigSearchDepth;
+            s_xmlDsigSearchDepth = 20;
             return s_xmlDsigSearchDepth.Value;
```

Comment:
This whole method could be `return 20;`.  The `int?` isn't required once it's not configurable.